src/sigal/gallery.py

```diff
@@ -286,6 +287,15 @@ def _get_markdown_metadata(self):
 
         return meta
 
+    @cached_property
+    def lat(self):
+        return self.markdown_metadata.get("lat", {})
```

Comment:
```suggestion
        return self.markdown_metadata.get("lat", "")
```
should return a string by default I think ?

Reply:
Or None which simplifies the code even further, allowing get() its default behaviour.
The related if statement in Jinja works the same.